voting/src/voting.rs

```diff
@@ -46,6 +76,38 @@ impl Context {
     fn meeting_mut(&mut self) -> &mut dyn GeneralMeetingManager {
         self.general_meeting.as_mut().unwrap().as_mut()
     }
+
+    fn excute_tx(&mut self, transaction: &Transaction) -> Result<H256, ExecuteError> {
+        match transaction.tx_type() {
+            VOTE_PAPER_TX_TYPE => {
+                let tx: PaperOwnTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| ExecuteError::InvalidFormat)?;
+                tx.verify().map_err(|_| ExecuteError::InvalidSignature)?;
+
+                let meeting_id = tx.tx.action.general_meeting_id;
+                let name = tx.tx.action.voter_name;
+                let shares = tx.tx.action.number_of_shares;
+                let agendas = tx.tx.action.agenda_number;
+                let voter_pub_key = tx.tx.action.voter_public_key;
+
+                let meeting: GeneralMeeting =
+                    self.meeting_mut().get_meeting(&meeting_id).map_err(|_| ExecuteError::InvalidMeeting)?;
```

Comment:
`get_meeting` returns error only when there is no meeting data using the meeting_id.
Let's change the error name more specific like "GeneralMeetingNotFound".